database/migrations/create_filament_redirects_table.php.stub

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create(config('filament-redirects.table_name'), function (Blueprint $table) {
+            $table->id();
+
+            $table->text('from_url');
+            $table->text('to_url');
+            $table->integer('status')->default(301);
```

Comment:
Consider using index on `from_url` since it will be used in where clause on every page request.